docs/platform/howto/add-authentication-method.rst

```diff
@@ -0,0 +1,14 @@
+Add authentication methods
+===========================
+
+You can authenticate directly with your email and password in the `Aiven Console <https://console.aiven.io/>`_ or use single sign-on through providers like GitHub, Google and Microsoft.
+
+To add an authentication method for your user account: 
```

Comment:
Are we using the **banished** word (`account`), in this context? :grin:

Reply:
In this context, yes (though sparingly to avoid any confusion).